package.json

```diff
@@ -12,7 +12,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "debug": "2.2.0",
-    "datadog-metrics": "^0.3.0"
+    "datadog-metrics": "^0.8.1"
```

Comment:
at least 0.8.1 is required to make sure 
1. count is used instead of counter as counter is deprecated in datadog 
2. count is not incremented for value 0